### PR TITLE
Always ensure reqs start on a new line

### PR DESCRIPTION
**Pull request by mxr**

### CODE REVIEW

File: setup_cfg_fmt.py [hunk 1]
```diff
@@ -198,15 +198,21 @@ def _python_requires(
...
+    else:
+        install_requires = [r.strip() for r in stripped.split(';')]
 
     normalized = sorted(
         (_normalize_req(req) for req in install_requires),
         key=lambda req: (';' in req, _req_base(req), req),
     )
 
-    if len(normalized) > 1:
+    if len(normalized) > 1 or ';' in normalized[0]:
```
Comment (asottile) [inline setup_cfg_fmt.py]: Maybe it sounds just always do this :thinking:

File: setup_cfg_fmt.py [hunk 2]
```diff
@@ -198,15 +198,21 @@ def _python_requires(
...
     raw = cfg.get('options', which, fallback='')
+    stripped = raw.strip()
+    if not stripped:
+        return []
 
-    install_requires = raw.strip().splitlines()
+    if raw.startswith(os.linesep):
+        install_requires = stripped.splitlines()
+    else:
+        install_requires = [r.strip() for r in stripped.split(';')]
```
Comment (asottile) [inline setup_cfg_fmt.py]: I wouldn't bother with this, AFAIK this weird semicolon behavior isn't documented anywhere

Comment (mxr) [reply setup_cfg_fmt.py]: OK

File: setup_cfg_fmt.py (hunk 1 above)
Comment (mxr) [reply setup_cfg_fmt.py]: sure, updated

**Comment by asottile** [comment]
I'll merge this when I'm at a computer and can release

**Pull request merged**